update-version.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
```

Comment:
we can maybe use `set -e` or `trap` to stop once non-zero error code is returned by any command (instead of continuing the execution)